### PR TITLE
✨ Add Pixi rule to check for Icon Font usage

### LINKED ISSUE #5325: Add Pixi rule for Icon Font Usage
### Problem
Icon fonts have a negative impact on CLS, causing layout shift as they load in (unless blocked from loading). Document authors are unaware of the impact since it rarely appears on high-speed connections.

### Solution
Identify when icon fonts are used in a document, (potentially via screenshot diffing or checking the content of each text node for known patterns). When detected present this as an opportunity to address CLS.

### Alternatives
1. Use screenshot diffing
2. Detect nodes with :before and known icon-font providers.
3. Provide confidence of violation when known, and just warn when uncertain.

### PR DESCRIPTION
**Pull request by lluerich**
Fixes #5325 by adding the linter rule defined in ampproject/amp-toolbox#1145 to the Pixi checks.

Draft until text is final.

### CODE REVIEW

**Review by matthiasrohmer — APPROVED** [review]
Looks fine implementation-wise 🙂

**Comment by lluerich** [comment]
@sebastianbenz are you fine with the provided copy?

File: pages/content/pixi/recommendations/no-icon-fonts.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+$title: Don't use icon fonts
+$order: 200
+tags:
+  - cls
+  - lcp
+---
+Icon fonts have a negative impact on CLS, causing layout shift as they load in (unless blocked from loading). Document authors are unaware of the impact since it rarely appears on high-speed connections.
```
Comment (sebastianbenz) [inline pages/content/pixi/recommendations/no-icon-fonts.md:8]: `Icon fonts negatively affect accessibility and load times of your page. They also can cause content shifts on load. Consider using SVGs as an alternative instead to improve LCP and CLS.`

**Review by sebastianbenz — APPROVED** [review]

File: pages/content/pixi/recommendations/no-icon-fonts.md (hunk 1 above)
Comment (lluerich) [reply pages/content/pixi/recommendations/no-icon-fonts.md:8]: I would move the ```also``` after the ```can```, so:
```... They can also cause content shifts on load.```

Comment (sebastianbenz) [reply pages/content/pixi/recommendations/no-icon-fonts.md:8]: Hm. Maybe we should make it even simpler:

```
Icon fonts negatively affect accessibility and performance of your page. Consider using SVGs as an alternative instead to improve LCP and CLS.
```

Comment (lluerich) [reply pages/content/pixi/recommendations/no-icon-fonts.md:8]: Perfect. 👍🏻

**Pull request merged**